prompts/prompts-larivasd.md

```
@@ -0,0 +1,109 @@
+# Prompts Utilizados - Pipeline GitHub Actions
+
+**IDE Utilizado:** Cursor
+
+## Prompts del Usuario
+
+1. **Conexión a EC2 y comandos incorrectos:**
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)